Documentation/Signoffs/SpeedometerSystem.md

```diff
@@ -0,0 +1,48 @@
+SPEEDOMETER SUBSYSTEM
+--------------------
+### Functions of the Subsystem
+The speedometer subsystem must receive output from the current driving simulation and display the speed the user is going. A microcontroller will be used to pull the speed data from the MATLAB code and display it on 4-digit a 7-segment display.
```

Comment:
I think we can do something better than a cheap, hideous seven segment. This should look and feel appropriate to the application aesthetically.